replicate/increment.go

```diff
@@ -163,6 +170,10 @@ func (c *consumer) getNewFiles(
 				// skip handling this file
 				return nil
 			}
+			// manifest
+			if err = c.GenManifestFile(ctx, path, size); err != nil {
+				return nil
+			}
```

Comment:
think about this case:
1. cdc generate 1.csv, 2.csv, ..., 5.csv
2. tidb2dw upload 1.manifest, 2.manifest, ..., 5.manifest
3. tidb2dw merge 1.csv, 2.csv, then 1.csv, 2.csv will be deleted
4. pause tidb2dw
5. restart tidb2dw, and tidb2dw will upload 3.manifest, 4.manifest and 5.manifest again.

And we can delete manifest after merge cdc file.